src/components/ListBoxMenu/ListBoxMenu.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import Icon from '../Icon';
+
+const handleOnClick = event => {
+  // Prevent default interaction here so that the event bubblers up to
+  // the `onClick` handler supplied by `getItemProps`
+  event.preventDefault();
+};
+
+const ListBoxMenu = ({
+  items,
+  selectItem,
+  selectedItem,
+  highlightedIndex,
+  itemToString,
+  getItemProps,
+}) => (
+  <div className="bx--list-box__menu">
+    {items.map((item, i) => (
+      <div
+        key={item.id}
+        className={cx({
+          'bx--list-box__menu-item': true,
+          'bx--list-box__menu-item--highlighted': highlightedIndex === i,
+          'bx--list-box__menu-item--active': selectedItem.indexOf(item) !== -1,
+        })}
+        {...getItemProps({
+          item,
+          index: i,
+        })}>
+        <input
+          id={item.id}
```

Comment:
@joshblack We need to think about multiple instances of `<MultiSelect>` - What if two of `<MultiSelect>`s have items with the same ID? We’d need to assign unique ID per instance (e.g. https://github.com/asudoh/carbon-components-react/blob/74c19d33f76a64428e97726f514b1e78bde786e6/src/components/ListBoxMenu/ListBoxMenu.js#L7-L11) to make the ID here unique. Thanks!

Reply:
Fixed! Just am going to use the id provided from `getItemProps` 😄